src/EFCore.PG.NodaTime/Query/ExpressionTranslators/Internal/NpgsqlNodaTimeMemberTranslatorPlugin.cs

```diff
@@ -38,7 +39,22 @@ namespace Npgsql.EntityFrameworkCore.PostgreSQL.NodaTime
     /// <remarks>
     /// See: https://www.postgresql.org/docs/current/static/functions-datetime.html
     /// </remarks>
-    public class NodaTimeMemberTranslator : IMemberTranslator
+    public class NpgsqlNodaTimeMemberTranslatorPlugin : IMemberTranslatorPlugin
+    {
+        public virtual IEnumerable<IMemberTranslator> Translators { get; }
+            = new IMemberTranslator[]
```

Comment:
As mentioned above, one of the above cases had `{ get } = SomeType[]`, and looked nice/terse with the reduced nesting.

Reply:
Done